test/models/user_test.rb

```diff
@@ -14,8 +14,7 @@ class UserTest < ActiveSupport::TestCase
       it "should require content in #{attribute}" do
         @user[attribute.to_sym] = nil
         assert_not @user.valid?
-        assert_equal "can't be blank",
-                     @user.errors.messages[attribute.to_sym].first
+        assert @user.errors.messages[attribute.to_sym].include? "can't be blank"
```

Comment:
side effect of not being able to use validatable :(, or possibly I copypasta'd wrong in `user.rb`.